recipe/meta.yaml

```
@@ -1,54 +1,49 @@
+{% set version = "5.32.1" %}
+{% set major_minor_version = version.split(".")[:2] | join(".") %}
 
 package:
   name: perl
-  version: {{ unix_version }}  # [not win]
-  version: {{ win_version }}   # [win]
+  version: {{ version }}
 
 source:
-  - url: http://www.cpan.org/src/5.0/perl-{{ unix_version }}.tar.xz                                                 # [unix]
-    sha256: 57cc47c735c8300a8ce2fa0643507b44c4ae59012bfdad0121313db639e02309                                        # [unix]
-    patches:                                                                                                        # [unix]
-      - 0001-Use-userelocatableinc-together-with-useshrplib.patch                                                   # [unix]
-      - 0002-Disable-direct-invocation-of-linker-if-LD-is-set.patch                                                 # [unix]
-  - url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-64bit-portable.zip  # [win64]
-    url: http://strawberryperl.com/download/{{ win_version }}/strawberry-perl-{{ win_version }}-32bit-portable.zip  # [win32]
-    sha1: fac226b31461f2392702822052a3a5628917f857                                                                  # [win64]
-    sha1: 28bca91cadd6651c2b2463db8587c170bf17f2fa                                                                  # [win32]
-    # sha1 hashes published at: http://strawberryperl.com/releases.html
+  - url: http://www.cpan.org/src/5.0/perl-{{ version }}.tar.xz
+    sha256: 57cc47c735c8300a8ce2fa0643507b44c4ae59012bfdad0121313db639e02309
+    patches:
+      - 0001-Use-userelocatableinc-together-with-useshrplib.patch  # [unix]
+      - 0002-Disable-direct-invocation-of-linker-if-LD-is-set.patch
+      - 0003-update-setting-for-conda-directory-structure.patch  # [win]
+      - 0004-skip-failing-Net-Ping-test-on-win.patch  # [win]
+      - 0005-skip-perlbug-test-that-fails-on-multiple-platforms.patch
+      - 0006-allow-t-TTY-to-fail-on-Windows.patch  # [win]
+      - 0007-skip-single-magic-test-on-non-AMD64-unix.patch # [unix]
+      - 0008-skip-newly-failing-Win32-tests.patch # [win]
 
 build:
-  number: 7
-  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_strawberry  # [win]
-  string: {{ PKG_BUILDNUM }}_h{{ PKG_HASH }}_perl5       # [not win]
+  skip: True  # [osx]
+  number: 8
   run_exports:
     weak:
-      - >-
-        {{ pin_subpackage("perl", max_pin="x.x") }}
-        *_strawberry  # [win]
-        *_perl5       # [not win]
+      - {{ pin_subpackage("perl", max_pin="x.x") }}
     noarch:
-      - >-
-        {{ pin_subpackage("perl", max_pin="x") }}
-        *_strawberry  # [win]
-        *_perl5       # [not win]
+      - {{ pin_subpackage("perl", max_pin="x") }}
 
 requirements:
   build:
     - {{ compiler('c') }}  # [unix]
-    - make                 # [unix]
-    - posix                # [win]
+    - m2w64-gcc            # [win]
+    - {{ stdlib("c") }}
```

```suggestion
    - {{ stdlib("c") }}           # [unix]
    - {{ compiler("m2w64_c") }}   # [win]
    - {{ stdlib("m2w64_c") }}     # [win]   
```
You'll want to set this instead to get the MinGW toolchain working here now that the UCRT migration has happened.

---

After re-render:
```
package m2w64-gcc-5.3.0-6 requires msys2-conda-epoch 20160418, but none of the providers can be installed
```

---

For local testing, use `conda build -m .ci_support/win64.yaml .` so that it uses the rendered build config that fills in the correct variables.